test/integration/internal-extension.js

```diff
@@ -100,14 +100,17 @@ test('load sync', t => {
 
     t.equal(vm.runtime._blockInfo.length, 1);
 
-    // blocks should be an array of two items: a button pseudo-block and a reporter block.
-    t.equal(vm.runtime._blockInfo[0].blocks.length, 2);
+    // blocks should be an array of two items: a button pseudo-block, a reporter block, and a command block.
+    t.equal(vm.runtime._blockInfo[0].blocks.length, 3);
     t.type(vm.runtime._blockInfo[0].blocks[0].info, 'object');
     t.type(vm.runtime._blockInfo[0].blocks[0].info.func, 'MAKE_A_VARIABLE');
     t.equal(vm.runtime._blockInfo[0].blocks[0].info.blockType, 'button');
     t.type(vm.runtime._blockInfo[0].blocks[1].info, 'object');
     t.equal(vm.runtime._blockInfo[0].blocks[1].info.opcode, 'exampleOpcode');
     t.equal(vm.runtime._blockInfo[0].blocks[1].info.blockType, 'reporter');
+    t.type(vm.runtime._blockInfo[0].blocks[2].info, 'object');
+    t.equal(vm.runtime._blockInfo[0].blocks[2].info.opcode, 'exampleDynamicOpcode');
+    t.equal(vm.runtime._blockInfo[0].blocks[2].info.blockType, 'command');
```

Comment:
Shouldn't this include a test for the context menu itself? Maybe something like this? (WARNING: not tested...)
```suggestion
    t.equal(vm.runtime._blockInfo[0].blocks[2].info.blockType, 'command');
    t.equal(vm.runtime._blockInfo[0].blocks[2].info.customContextMenu.length, 1);
    t.type(vm.runtime._blockInfo[0].blocks[2].info.customContextMenu[0], 'function');
```

Reply:
Done!